plugins/pulp_python/plugins/models.py

```diff
@@ -183,48 +180,9 @@ def _metadata_label(attribute):
         label = attribute[0].upper() + attribute[1:]
         return label.replace('_', '-')
 
-    def __init__(self, name, version, summary, home_page, author, author_email, license,
-                 description, platform, _filename, _checksum, _checksum_type, _metadata_file):
-        """
-        Initialize self with the given parameters as its attributes.
-
-        :param name:           The name of the package.
-        :type  name:           basestring
-        :param version:        The package's version.
-        :type  version:        basestring
-        :param summary:        A paragraph summarizing the package.
-        :type  summary:        basestring
-        :param home_page:      A URL for the package's website.
-        :type  home_page:      basestring
-        :param author:         The author's name.
-        :type  author:         basestring
-        :param author_email:   The author's e-mail address.
-        :type  author_email:   basestring
-        :param license:        The package's license.
-        :type  license:        basestring
-        :param description:    A description of the package.
-        :type  description:    basestring
-        :param platform:       A list of platforms that the package is intended to be used on.
-        :type  platform:       basestring
-        :param _filename:      The package filename.
-        :type  _filename:      basestring
-        :param _checksum:      The checksum of the package.
-        :type  _checksum:      basestring
-        :param _checksum_type: The name of the algorithm used to calculate the checksum.
-        :type  _checksum_type: basestring
-        :param _metadata_file: The path of the metadata file in the package
-        :type  _metadata_file: basestring
-        """
-        for attr in self._ATTRS:
-            setattr(self, attr, locals()[attr])
-
-        self._unit = None
-
     def __repr__(self):
         """
-        Return a string representation of self.
-
         :return: A string representing self.
         :rtype:  basestring
         """
-        return 'Python Package: %(name)s-%(version)s' % {'name': self.name, 'version': self.version}
+        return 'Package(name={}, version={})'.format(self.name, self.version)
```

Comment:
Good thought. I do like sticking to the guidance on keeping ``repr`` close to something that could be passed to ``eval``, but something more user-friendly would be good in a ``__str__`` method.